coral-component-masonry/examples/index.html

```diff
@@ -130,89 +138,95 @@ <h2 class="coral-Heading--M">Usage notes</h2>
           <a href="#" class="coral-Link append" data-count="100">append 100</a> |
         </label>
         <i>(double click on item to remove it)</i>
-
-        <coral-masonry orderable id="masonry" layout="fixed-centered" columnwidth="250" orderable>
-          <coral-masonry-item class="coral-Well">
-            <article>
-              <span coral-masonry-draghandle></span>
-              With image
-              <img src="http://via.placeholder.com/600x300" alt="">
-            </article>
-          </coral-masonry-item>
-          <coral-masonry-item class="coral-Well">
-            <article>
-              <span coral-masonry-draghandle></span>
-              card
-              <br>card
-            </article>
-          </coral-masonry-item>
-          <coral-masonry-item class="coral-Well" colspan="2">
-            <article>
-              <span coral-masonry-draghandle></span>
-              card
-              <br>card
-              <br>card
-            </article>
-          </coral-masonry-item>
-          <coral-masonry-item class="coral-Well">
-            <article>
-              <span coral-masonry-draghandle></span>
-              card
-              <br>card
-              <br>card
-              <br>card
-            </article>
-          </coral-masonry-item>
-          <coral-masonry-item class="coral-Well" colspan="2">
-            <article>card
-              <br>card
-              <br>card
-              <br>card
-              <br>card</article>
-          </coral-masonry-item>
-          <coral-masonry-item class="coral-Well">
-            <article>card
-              <br>card
-              <br>card
-              <br>card
-              <br>card
-              <br>card</article>
-          </coral-masonry-item>
-          <coral-masonry-item class="coral-Well">
-            <article>card
-              <br>card</article>
-          </coral-masonry-item>
-          <coral-masonry-item class="coral-Well" colspan="2">
-            <article>card
-              <br>card
-              <br>card</article>
-          </coral-masonry-item>
-          <coral-masonry-item class="coral-Well">
-            <article>card
-              <br>card
-              <br>card
-              <br>card</article>
-          </coral-masonry-item>
-          <coral-masonry-item class="coral-Well" colspan="2">
-            <article>card
-              <br>card
-              <br>card
-              <br>card
-              <br>card</article>
-          </coral-masonry-item>
-          <coral-masonry-item class="coral-Well">
-            <article>card
-              <br>card
-              <br>card
-              <br>card
-              <br>card
-              <br>card</article>
-          </coral-masonry-item>
-        </coral-masonry>
+        <div>
```

Comment:
@jasghar Wrap `coral-masonry` in a container so that `ariaGrid` has a parentElement that does not contain additional children.